cmd/podman/common/create_opts.go

```diff
@@ -11,7 +14,7 @@ type ContainerCLIOpts struct {
 	CapAdd            []string
 	CapDrop           []string
 	CGroupsNS         string
-	CGroups           string
+	CGroupsMode       string
```

Comment:
this variable name cannot be changed. it correlates to the create.go line 58 and is the pointer endpoint for where the cgroups user input is stored.

Reply:
Where is line 58?

Reply:
The name CGroups is too confusing for me at least.

Reply:
as i said, create.go ... you just gotta change the variable there too if you want to rename it